pupil_src/shared_modules/calibration_choreography/natural_feature_plugin.py

```diff
@@ -27,7 +27,8 @@
     ChoreographyMode,
     ChoreographyAction,
 )
-from gaze_mapping import Gazer2D_v1x, Gazer3D_v1x
+from gaze_mapping import Gazer2D
+from gaze_mapping import Gazer3D_v1x
```

Comment:
Why are these even in here? They're not being used, are they?
Same below in `screen_marker_plugin.py`